CMakeLists.txt

```diff
@@ -323,14 +323,13 @@ else()
   add_subdirectory(3party/icu)
 endif()
 
-set(JANSSON_EXAMPLES OFF)
 set(JANSSON_BUILD_DOCS OFF)
-set(JANSSON_WITHOUT_TESTS ON)
+set(JANSSON_BUILD_MAN OFF)
+set(JANSSON_EXAMPLES OFF)
 set(JANSSON_INSTALL OFF)
+set(JANSSON_WITHOUT_TESTS ON)
 find_package_or_fallback_to_3party(Jansson 2.14.0 3party/jansson/jansson/)
```

Comment:
This line would be as good as unconditional dead code if it is impossible to find the package because of the disabled FindJansson.cmake. Why not put it behind an opt-in feature flag for the given 3party?